src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/EventsSpec.scala

```diff
@@ -9,10 +9,11 @@ import akka.http.scaladsl.model.sse.ServerSentEvent
 import akka.http.scaladsl.model.{HttpRequest => Req, _}
 import akka.stream.alpakka.sse.scaladsl.EventSource
 import akka.stream.scaladsl.Sink
-import ch.epfl.bluebrain.nexus.rdf.syntax._
 import ch.epfl.bluebrain.nexus.commons.http.JsonLdCirceSupport._
```

Comment:
need to drop all dependencies on `*.nexus.commons.*` as it won't be available in the nexus repo